spec/css/functions/special.hrx

```diff
@@ -25,3 +25,187 @@ a {b: -a-calc(#{0})}
+
+<===> comment/element/after_open_paren/loud/output.css
+a {
+  b: element(/**/ c);
+}
+
+<===>
+================================================================================
+<===> comment/element/after_open_paren/silent/input.scss
+a {
+  b: element(//
+    c);
+}
+
+<===> comment/element/after_open_paren/silent/output.css
+a {
+  b: element( c);
+}
+
+<===>
+================================================================================
+<===> comment/element/before_close_paren/loud/input.scss
+a {b: element(c /**/)}
+
+<===> comment/element/before_close_paren/loud/output.css
+a {
+  b: element(c /**/);
+}
+
+<===>
+================================================================================
+<===> comment/element/before_close_paren/silent/input.scss
+a {
+  b: element(c //
+    );
+}
+
+<===> comment/element/before_close_paren/silent/output.css
+a {
+  b: element(c  );
+}
+
+<===>
+================================================================================
+<===> comment/expression/after_open_paren/loud/input.scss
+a {b: expression(/**/ c)}
+
+<===> comment/expression/after_open_paren/loud/output.css
+a {
+  b: expression(/**/ c);
+}
+
+<===>
+================================================================================
+<===> comment/expression/after_open_paren/silent/input.scss
+a {
+  b: expression(//
+    c);
+}
+
+<===> comment/expression/after_open_paren/silent/output.css
+a {
+  b: expression( c);
+}
+
+<===>
+================================================================================
+<===> comment/expression/before_close_paren/loud/input.scss
+a {b: expression(c /**/)}
+
+<===> comment/expression/before_close_paren/loud/output.css
+a {
+  b: expression(c /**/);
+}
+
+<===>
+================================================================================
+<===> comment/expression/before_close_paren/silent/input.scss
+a {
+  b: expression(c //
+    );
+}
+
+<===> comment/expression/before_close_paren/silent/output.css
+a {
+  b: expression(c  );
+}
+
+<===>
+================================================================================
+<===> comment/progid/after_open_paren/loud/input.scss
+a {b: progid:c(/**/ d)}
+
+<===> comment/progid/after_open_paren/loud/output.css
+a {
+  b: progid:c(/**/ d);
+}
+
+<===>
+================================================================================
+<===> comment/progid/after_open_paren/silent/input.scss
+a {
+  b: progid:c(//
+    d);
+}
+
+<===> comment/progid/after_open_paren/silent/output.css
+a {
+  b: progid:c( d);
+}
+
+<===>
+================================================================================
+<===> comment/progid/before_close_paren/loud/input.scss
+a {b: progid:c(d /**/)}
+
+<===> comment/progid/before_close_paren/loud/output.css
+a {
+  b: progid:c(d /**/);
+}
+
+<===>
+================================================================================
+<===> comment/progid/before_close_paren/silent/input.scss
+a {
+  b: progid:c(d //
```

Comment:
If there are multiple lines with `//`, are they considered a single silent comment, or multiple silent comments?

For example, would this block:
```scss
a {
  b: progid:c(d // foo
                // bar
    );
}
```
result in this?:
```css
a {
  b: progid:c(d

    );
}
```
And if so, is that OK?

Reply:
They are not considered a single comment, but the whitespace around them is still considered "adjacent" so you won't end up with multiple newlines.